src/array/linestring/array.rs

```diff
@@ -130,7 +130,7 @@ impl<'a, O: OffsetSizeTrait> GeometryArrayTrait<'a> for LineStringArray<O> {
             "ARROW:extension:name".to_string(),
             "geoarrow.linestring".to_string(),
         );
-        Arc::new(Field::new("", self.storage_type(), true).with_metadata(field_metadata))
+        Arc::new(Field::new("geometry", self.storage_type(), true).with_metadata(field_metadata))
```

Comment:
I'm reluctant to call this `"geometry"` because we don't know the actual name of the field referring to the data in this column (and e.g. there could be two geometry columns in one table). I might switch this to return the field metadata directly instead of a full `FieldRef` (ref https://github.com/geoarrow/geoarrow-rs/issues/218)